tools/testing/fetch_mailman.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+FALCON_ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )/../.." &> /dev/null && pwd )
+MAILMAN_PATH=$FALCON_ROOT/.ecosystem/mailman
+
+# TODO(vytas): Detect the latest version from git tags or PyPi JSON.
+# NOTE(vytas): One approach: `curl -Ls https://pypi.org/pypi/mailman/json | jq -r .info.version`
+# MAILMAN_VERSION=3.3.5
+
+# Clean up in case we are running locally and not in CI
+rm -rf $MAILMAN_PATH
+
+mkdir -p .ecosystem
+git clone https://gitlab.com/mailman/mailman.git/ $MAILMAN_PATH
```

Comment:
It would be good to do something like `--depth 100` or something, 10k commits can be expensive to clone on every CI run and isn't really needed.

Reply:
Ah, right. Since I'm not checking out any tag as I intended, even a depth of 1 would be enough :)